content/introducing-user-roles-authorization-with-supertokens/index.md

```diff
@@ -0,0 +1,200 @@
+---
+title: "Introducing User Roles Authorization with SuperTokens"
+description: "With user roles, you can now use Supertokens to easily attach a different set of permissions to each user."
+date: "2022-10-27"
+cover: "user_roles_cover.png"
+category: "featured"
+author: "SuperTokens team"
+---
+
+### Table of Contents
+- [Part 1 - Introducing user roles and understanding why authorization matters](#part-1---introducing-user-roles-and-understanding-why-authorization-matters)
```

Comment:
it's very strange that you have part 1 and part 2 when the content of both of these are in this blog. Please remove "part 1" "part 2"